src/layers/routes/layersRouterFactory.ts

```diff
@@ -8,6 +8,7 @@ const layersRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
 
   router.get('/', controller.getLayer);
   router.post('/', controller.addLayer);
+  router.post('/best', controller.addLayerToBest);
```

Comment:
we said we will remove the term "best" from our code, no?

Reply:
fixed in ee20559